test/_socketStream.js

```diff
@@ -1,3 +1,4 @@
+'use strict';
```

Comment:
The same as above, please revert this change as it's not obvious why this would be changed in the same commit as a dependency version bump.